modules/ROOT/pages/salesforce/salesforce-connector-100-config-topics.adoc

```diff
@@ -316,31 +294,23 @@ output application/java
 [[usagenotes]]
 == Usage Notes
 
-=== Fields To Null
-
-The configurations have a checkbox called Can Clear Fields by Updating Field Value to Null. If checked, all the fields in a request that have a Null value are added to the `fieldsToNull` field and sent to Salesforce.
-
-You can decide which fields to set to Null without being forced to use the `fieldsToNull` field.
-
 === Upsert
 
-Unless you configure the External ID Field Name for the sObject to which you are trying to upsert, every use of the upsert fails.
+Unless you configure the External ID Field Name for the sObject to which you are trying to upsert, the Upsert operation fails.
 
-The upsert operation does not work with the sObject `priceBookentry2`.
+The Upsert operation does not work with the sObject `priceBookentry2`.
```

Comment:
What is the common noun object for "the sObject `priceBookentry2`"?

Reply:
This is a Salesforce API term. I am leaving as is.